cpp/tests/io/nested_json_test.cpp

```diff
@@ -588,11 +588,11 @@ TEST_P(JsonParserTest, ExpectFailMixStructAndList)
     R"( [{"a":[123, {"0": 123}], "b":1.0}, {"b":1.1}, {"b":2.1}] )",
     R"( [{"a":[123, "123"], "b":1.0}, {"b":1.1}, {"b":2.1}] )"};
 
+  // We do not currently support a mix of lists and structs.
```

Comment:
Trying to incrementally cut down on the usage of "we". (We, the cudf developer community, are separate from the product, libcudf.)
```suggestion
  // A mix of lists and structs is not currently supported.
```

Reply:
I kept it active voice, "libcudf does not ..."